src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.Tests/Microsoft.DotNet.SourceBuild.Tests.csproj

```diff
@@ -62,6 +62,9 @@
       <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).CustomPackagesPath">
         <Value>$(SourceBuildTestsCustomSourceBuiltPackagesPath)</Value>
       </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).CustomPackagesPathIsInclusive">
+        <Value>$(SourceBuildTestsCustomSourceBuiltPackagesPathIsInclusive)</Value>
```

Comment:
Can you please add a comment describing this property? I assume it means that the packages path contains everything needed for this test library?

Reply:
This property has been removed as part of https://github.com/dotnet/sdk/pull/48327/commits/5a869520be05ed8f89bf974f21b854d699618a7a